redwood-layout-uiview/src/commonMain/kotlin/app/cash/redwood/layout/uiview/UIViewFlexContainer.kt

```diff
@@ -31,26 +31,32 @@ import platform.darwin.NSInteger
 internal class UIViewFlexContainer(
   direction: FlexDirection,
 ) : YogaFlexContainer<UIView>, ChangeListener {
+  private val modifiers = mutableListOf<Modifier>()
   private val yogaView: YogaUIView = YogaUIView(
     applyModifier = { node, index ->
-      node.applyModifier(children.widgets[index].modifier, Density.Default)
+      node.applyModifier(modifiers[index], Density.Default)
```

Comment:
I'm a little unclear on this. Wasn't the previous code looking at the individual widgets modifiers? but are the modifiers we're saving into this mutableList() the same here.

Reply:
Yeah, we’re now keeping our own private copy of the modifiers because keeping a reference to the Widget creates a retain cycle.